pandas/core/ops.py

```diff
@@ -1327,6 +1327,10 @@ def wrapper(self, other, axis=None):
 
         res_name = get_op_result_name(self, other)
 
+        if isinstance(other, list):
```

Comment:
why is ``is_list_like`` (maybe after some other comparisons) enough here?

Reply:
ATM the `isinstance(other, list)` check is done below the `isinstance(other, (np.ndarray, pd.Index))` check.  Wrapping lists earlier let us send lists through that same ndarray/Index block.  Ideally the catchall `else:` block can be reduced to only-scalars, but we're not there yet.